packages/cubejs-databricks-jdbc-driver/src/post-install.ts

```diff
@@ -2,11 +2,15 @@ import 'source-map-support/register';
 
 import { displayCLIError } from '@cubejs-backend/shared';
 
+import fs from 'fs';
+import path from 'path';
 import { downloadJDBCDriver } from './installer';
 
 (async () => {
   try {
-    await downloadJDBCDriver(true);
+    if (!fs.existsSync(path.join(__dirname, '..', 'download', 'SparkJDBC42.jar'))) {
```

Comment:
📓 don't reload on every yarn install

Reply:
This block could be called more than onece, which could be a cause for a second download. You should check not only the file existing, but also a promise of the download function. Isn't it?

Reply:
Didn't get the point, in which case we can call it more than once? Two parallel yarn installs looks like a redundant case or do you mean smth else?